models/articles.model.js

```diff
@@ -41,3 +39,51 @@ exports.updateArticleVotes = (article_id, votes = 0) => {
       }
     });
 };
+
+exports.selectArticles = topicFilter => {
+  if (topicFilter) {
+    return db
+      .query(
+        `SELECT articles.article_id, title, topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comment_id) ::INT AS comment_count
+      FROM articles
+          LEFT JOIN comments ON articles.article_id = comments.article_id
+      WHERE topic = $1
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at`,
+        [topicFilter]
+      )
+      .then(({ rows }) => {
+        return rows;
+      });
+  } else {
```

Comment:
This isn't very dry code as we are repeating a lot of our db.query string. Would it not be possible to just amend the query string based upon whether the topicFilter exists?